mrjones2014/yaclt
# add pre- and post- command hooks

Resolves: #76 

# How to Test

1. Run `yarn build`
2. Update `yactlrc.js` and add the following
```
preHook: (commandName) => commandName && console.log(commandName)
```
3. Run `./dist/index.js validate`
4. It should fail because the `preHook` returned falsy value
5. Update `yacltrc.js` and update the hook to
```
preHook: (commandName) => commandName
```
6. Run `./dist/index.js validate` 
7. It should succeed because the hook returns a truthy value
